code/game/objects/items/devices/traitordevices.dm

```diff
@@ -69,8 +69,9 @@ effective or pretty fucking useless.
 */
 
 /obj/item/healthanalyzer/rad_laser
-	custom_materials = list(/datum/material/iron=400)
-	var/irradiate = 1
+	var/irradiate = TRUE
+	var/stealth = FALSE
+	var/used = FALSE // is it cooling down?
```

Comment:
these don't need to be here since theyre already at the bottom of the definition